lib/components/Button.jsx

```diff
@@ -56,7 +56,7 @@ const getTextSizeClasses = (textSizeClasses, isText, size) => {
   }
 
   if (isText) {
-    `text-sm sm:text-base lg:text-2xl`
+    ;`text-sm sm:text-base lg:text-2xl`
```

Comment:
Am I crazy or what is going on here? 🤯

Reply:
Ah it was missing a return statement so prettier had no idea what was going on